docs/products/redis/concepts/restricted-redis-commands.rst

```diff
@@ -0,0 +1,20 @@
+Restricted Aiven for Redis®* commands 
+======================================
+
+To maintain the stability and security of the Redis environment, the Aiven for Redis* service restricts certain Redis commands. This section aims to provide you with a list of disabled/restricted commands.
+
+
+Disabled commands
+------------------
```

Comment:
@harshini-rangaswamy sorry my communication wasnt clear. We should include all the command under Admin and Restricted and put it under Disabled. 

We do not mentioned the why such command was blocked (e.g. commands under Admin is only made for Aiven, customers dont need to know this, from their perspective, those commands are blocked)

Reply:
@hoang-vo Update the content, please review this/